lua/nvim-treesitter/incremental_selection.lua

```diff
@@ -138,16 +138,13 @@ function M.attach(bufnr)
   local config = configs.get_module "incremental_selection"
   for funcname, mapping in pairs(config.keymaps) do
     if mapping then
-      ---@type string, string|function
-      local mode, rhs
-      if funcname == "init_selection" then
-        mode = "n"
-        ---@type function
-        rhs = M[funcname]
-      else
-        mode = "x"
-        rhs = M[funcname] ---@type function
+      local mode = funcname == "init_selection" and "n" or "x"
+      local rhs = M[funcname] ---@type function
+      
+      if not rhs then
+        utils.notify("Unknown keybinding: " .. funcname)
       end
+      
       vim.keymap.set(
```

Comment:
the keymap.set should be in the else block

Reply:
I thought about it, since the `vim.keymap.set` would fail anyway. That being said I think that the full stack trace is helpful? I don't have strong feelings about it though.